examples.md

```diff
@@ -277,6 +277,20 @@ The yarn cache directory will depend on your operating system and version of `ya
       ${{ runner.os }}-yarn-
 ```
 
+
+## Node - Pnpm
+
+```yaml
+  - name: Cache .pnpm-store
+    uses: actions/cache@v2
+    with:
+      path: |
+      	 ~/.pnpm-store
+         D:\.pnpm-store
+      key: ${{ runner.os }}-${{ hashFiles('**/pnpm-lock.yaml') }}```
```

Comment:
`pnpm-store` is mostly independent of `pnpm-lock.yaml`, and restoring the store even when the lock file has changed can save a lot of time

```suggestion
      key: ${{ runner.os }}```
```